pandas/meson.build

```diff
@@ -0,0 +1,51 @@
+incdir_numpy = run_command(py,
+  [
+    '-c',
+    'import os; os.chdir(".."); import numpy; print(numpy.get_include())'
```

Comment:
This does not work for me because it returns an absolute path. I have to modify it to `'import os; import numpy; print(os.path.relpath(numpy.get_include()))'`

Reply:
@dnicolodi is right, you'll want to change this to exactly what it is in SciPy: https://github.com/scipy/scipy/blob/a077de40998263544c2670c05a84b2414a1df9e9/scipy/meson.build#L30-L73, to ensure things like in-tree venvs are working smoothly.